pymeasure/instruments/hp/hp34401A.py

```diff
@@ -23,24 +23,24 @@
 #
 
 from pymeasure.instruments import Instrument
+from pymeasure.instruments.validators import strict_discrete_set
 
 
 class HP34401A(Instrument):
     """ Represents the HP 34401A instrument.
     """
 
-    voltage_dc = Instrument.measurement("MEAS:VOLT:DC? DEF,DEF", "DC voltage, in Volts")
+    FUNCTIONS = {"DCV": "VOLT", "DCV_RATIO": "VOLT:RAT",
+                 "ACV": "VOLT:AC", "DCI": "CURR", "ACI": "CURR:AC",
+                 "R2W": "RES", "R4W": "FRES", "FREQ": "FREQ",
+                 "PERIOD": "PER", "CONTINUITY": "CONT", "DIODE": "DIOD"}
 
-    voltage_ac = Instrument.measurement("MEAS:VOLT:AC? DEF,DEF", "AC voltage, in Volts")
```

Comment:
Why do you remove these properties?

Consider to mark them as deprecated in order to keep the instrument working for users, until they adjust their code.

Feel free to ask how to implement depreciation.

Reply:
I use a get_process now which emits a deprecation warning to minimize code duplication. Is it ok like that or do you have another idea?

Reply:
I like that idea to have a warning as a side effect. I marked, how to raise a proper warning.